collector/info_schema_auto_increment.go

```diff
@@ -0,0 +1,68 @@
+// Scrape auto_increment column information.
+
+package collector
+
+import (
+	"database/sql"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	infoSchemaAutoIncrementQuery = `
```

Comment:
No need for `const ()` blocks for just one const IMO. You could just do:

`const infoSchemaAutoIncrementQuery = ...`